editor/project_converter_3_to_4.cpp

```diff
@@ -3031,9 +3030,9 @@ Vector<String> ProjectConverter3To4::check_for_rename_gdscript_functions(Vector<
 	return found_renames;
 }
 
-// TODO, this function should run only on all .gd files and also on lines in tscn files which
+// TODO, this function should run only on all ".gd" files and also on lines in ".tscn" files which
```

Comment:
I would end this sentence with “and also on lines in ".tscn" files which are parts of built-in Scripts.”